app/src/main/java/org/simple/clinic/bloodsugar/history/adapter/BloodSugarHistoryListItem.kt

```diff
@@ -66,6 +63,25 @@ sealed class BloodSugarHistoryListItem : PagingItemAdapter.Item<Event> {
       holder.editButton.visibleOrGone(isBloodSugarEditable)
     }
 
+    private fun renderBloodSugarLevel(holder: ViewHolderX, context: Context, reading: BloodSugarReading) {
+      when {
+        reading.isLow -> {
+          holder.bloodSugarLevelTextView.visibility = View.VISIBLE
+          holder.bloodSugarLevelTextView.text = context.getString(R.string.bloodsugar_level_low)
+          holder.bloodSugarIconImageView.setImageResource(R.drawable.ic_blood_sugar_filled)
```

Comment:
Can you link the Figma / Zeplin link for this screen?

Reply:
Here you go: https://www.figma.com/file/vmZlmlFSCOrEYzTMkqsNB8/Simple-Production?node-id=7158%3A247